src/pages/Calculator.tsx

```diff
@@ -174,26 +174,20 @@ export const Calculator = (): React.ReactElement => {
     </button>
   )
 
+  const oneWeekOfMilliseconds = 7 * 24 * 60 * 60 * 1000
+  const currentTimeMillis = new Date().getTime()
+  const prevalenceDataDate = calculatorData.prevalanceDataDate
+  const prevalenceDataTimeMillis = prevalenceDataDate?.getTime()
+  const prevalenceIsStale =
+    prevalenceDataTimeMillis !== null &&
+    currentTimeMillis - prevalenceDataTimeMillis > oneWeekOfMilliseconds
+
+  const showStaleWarning = prevalenceIsStale && !suppressStaleWarning
+
   return (
     <div id="calculator">
       <Row>
         <Col md="12" lg="8" id="calculator-introduction">
-          {showWarning && (
```

Comment:
If you want me to keep this block around for easy drop-in general warnings in the future, just say the word.  Maybe `showGeneralWarning` if so to clarify?  🤷